PatientUpdate.cls

```diff
@@ -0,0 +1,13 @@
+public class PatientUpdate {
```

Comment:
This is class that handles PAyment trigger, so pls call it PaymentTriggerHandler